resources/instructions/createSObjectLwcQuickActions.html

```diff
@@ -1,81 +1,110 @@
-                </tr>
-                <tr>
-                    <td>Account</td>
-                    <td>✅</td>
-                </tr>
-                <tr>
-                    <td>Opportunity</td>
-                    <td>❌</td>
-                </tr>
-            </tbody>
-        </table>
-
-        <label
-            style="display: block; margin-top: 10px; margin-bottom: 8px"
-            for="generateLwcQuickActionsButton"
-        >
-            Click 'Generate LWC Quick Actions' to create the missing LWC Quick
-            Actions:
-        </label>
-        <button id="generateLwcQuickActionsButton">
-            Generate LWC Quick Actions
-        </button>
-        <script>
-            const generateQuickActionsButtonElement = document.getElementById(
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Offline Starter Kit: Create sObject LWC Quick Actions</title>
+</head>
+<style>
+    h1 {
+        font-size: 16px;
+    }
+
+    h2 {
+        font-size: 14px;
+    }
+
+    table {
+        border-collapse: collapse;
+    }
+
+    table,
+    th,
+    td {
+        border-width: 1px;
+        border-style: solid;
+    }
+
+    th,
+    td {
+        text-align: center;
+    }
+</style>
+
+<body>
+    <h1>Create sObject LWC Quick Actions</h1>
+
+    <p>
+        The following sObjects are present in your configured landing page:
+    </p>
+
+    <table id="quickActionStatusTable">
+        <thead>
+            <tr>
+                <th rowspan="2">sObject</th>
+                <th colspan="3">LWC Quick Actions</th>
+            </tr>
+            <tr>
+                <th>view</th>
+                <th>edit</th>
+                <th>create</th>
+            </tr>
+        </thead>
+        <tbody>
+            <!-- will be filled in programatically below -->
+        </tbody>
+    </table>
+
+    <label style="display: block; margin-top: 10px; margin-bottom: 8px" for="generateLwcQuickActionsButton">
+        Click 'Generate LWC Quick Actions' to create the missing LWC Quick
+        Actions:
+    </label>
+    <button id="generateLwcQuickActionsButton">
+        Generate LWC Quick Actions
+    </button>
+    <script>
+        const generateQuickActionsButtonElement = document.getElementById(
+            'generateLwcQuickActionsButton'
+        );
+        generateQuickActionsButtonElement.addEventListener('click', () => {
+            webviewMessaging.sendMessageRequest(
                 'generateLwcQuickActionsButton'
             );
-            generateQuickActionsButtonElement.addEventListener('click', () => {
-                webviewMessaging.sendMessageRequest(
-                    'generateLwcQuickActionsButton'
-                );
-            });
-        </script>
-        <script src="--- MESSAGING_SCRIPT_SRC ---"></script>
-    </body>
-</html>
+        });
+
+        // Wait until all scripts are loaded, before engaging with e.g.
```

Comment:
This is the meat of the change here -- makes a call to new function and dynamically generates table rows.